src/App.tsx

```diff
@@ -410,6 +415,44 @@ function App() {
                   </div>
                 </div>
 
+                {/* Collapsible section for organizer email */}
+                <CollapsibleSection
+                  initiallyOpen={!!form.creatorEmail}
+                  title={
+                    <span className="text-sm">
+                      Organizer email (optional)
+                    </span>
+                  }
+                >
+                  <Alert
+                    color="primary"
+                    variant="faded"
+                    title={
+                      <div className="mb-2 font-bold">
+                        Add an organizer email
+                      </div>
+                    }
+                    className="my-4 text-left"
+                    description={
+                      <div>
+                        We will use this email to enable RSVP functionality for attendees.
+                        <br />
+                        Attendees can respond with Yes, No, or Maybe to your event.
```

Comment:
I would change the description to:
"Attendees will see your email address and can RSVP with **Yes**, **No**, or **Maybe**, as well as send you a direct message.."